apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -172,3 +189,47 @@ function ProgressBar(partial: number | null, total: number | null) {
     </div>
   `;
 }
+
+function GraderAssignmentModal({
+  csrfToken,
+  courseStaff,
+}: {
+  csrfToken: string;
+  courseStaff: User[];
+}) {
+  return Modal({
+    id: 'grader-assignment-modal',
+    title: 'Assign instances to graders',
+    body: html`
+      <p>Assign instances to the following graders:</p>
+      ${courseStaff.map(
+        (staff) => html`
+          <div class="form-check">
+            <input
+              type="checkbox"
+              id="grader-assignment-${staff.user_id}"
+              name="assigned_grader"
+              value="${staff.user_id}"
+              class="form-check-input"
+            />
+            <label class="form-check-label" for="grader-assignment-${staff.user_id}">
+              ${staff.name ? `${staff.name} (${staff.uid})` : staff.uid}
+            </label>
+          </div>
+        `,
+      )}
+      <div class="mt-2 small alert alert-info">
```

Comment:
Nit, improved spacing:

```suggestion
      <div class="mt-3 mb-0 small alert alert-info">
```